source/examples/rapids-sagemaker-higgs/notebook.ipynb

```diff
@@ -1002,7 +1002,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "aws ecr delete-repository --force --repository-name"
+    "!aws ecr delete-repository --force --repository-name"
```

Comment:
This is a shell command but was being parsed as Python code because of the missing `!`. So yay, that's a legit bug `ruff` caught 🎉